en/computer_vision/obstacle_avoidance.md

```diff
@@ -0,0 +1,120 @@
+# Obstacle Avoidance
+
+*Obstacle Avoidance* enables a vehicle to navigate around obstacles when following a preplanned path.
+This feature is intended for automatic modes where an endpoint destination is known. 
+It is currently supported for multicopter vehicles in [Missions](#mission_mode) and [Offboard mode](#offboard_mode).
+
+This topic explains how the feature is set up and enabled in both modes.
+
+> **Tip** Manual modes instead use the much simpler *Collision Avoidance* approach, which simply stops the vehicle from crashing into obstacles.
```

Comment:
This is partially true. As you have written, we have enabled this interface only in AUTO modes. However since the `vehicle_trajectory_waypoint_desired` contains the desired velocity setpoints (generated by any FlightTask), it can be use for full obstacle avoidance in manual mode. I have tested the interface with such a system. However the user would need to modify [this](https://github.com/PX4/Firmware/blob/master/src/modules/mc_pos_control/mc_pos_control_main.cpp#L1187-L1188) and the rate at which the messages are sent from the autopilot. Plus it would need to use an obstacle avoidance algorithm that accepts as inputs velocity setpoints (both local and global planner don't support it). 
I am not sure if it's worth mentioning since it isn't fully supported.

Reply:
The main reason I added this at all was because collision avoidance and obstacle avoidance are so similar in name I felt we needed to differentiate. I think I will remove this though, and have the differentiation in a higher level "computer vision" page. 

I wouldn't mention this possibility in either case unless it was actually enabled. In addition, while it might be cool, I suspect that for manual control it would have little real additional value.